Modix.Services/Configuration/ConfigurationService.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Nett;
+
+namespace Modix.Services.Configuration
+{
+    public interface IConfigurationService
+    {
+        DiscordBotConfiguration LoadDiscordBotConfiguration();
+    }
+
+    public class ConfigurationService : IConfigurationService
+    {
+        private const string DiscordBotConfigurationPath = "configuration.tml";
+
+        public DiscordBotConfiguration LoadDiscordBotConfiguration()
+        {
+            return Toml.ReadFile<DiscordBotConfiguration>(DiscordBotConfigurationPath);
```

Comment:
Will this explode if the file doesn't exist?

Reply:
Yes, I'm not too sure how that wants to be handled, seeing as its a configuration that will supposedly drive a lot of behaviour.

Reply:
Probably throw an explicit exception for the configuration missing.

Reply:
This seems like something we could pick a sensible default for. We could just check if the file exists and load it if it is, otherwise return a default instance that always returns `false` for `PurgeInvites`.

Reply:
If we take that approach we should log when the file was not found and the default instance was returned.

Reply:
considering this is something that should be present in the container when we deploy, i'd rather fail the deploy than hum along with features disabled that we thought were active.

Reply:
That's what the logging comment was meant to address, but a fail-fast approach works, too. We should get a default one added to the repository to make sure it doesn't explode when working locally.